doc/src/build-instructions.md

```diff
@@ -140,36 +140,44 @@ Change directories to the `src/platforms/esp32` directory under the AtomVM sourc
 	shell$ cd <atomvm-source-tree-root>
 	shell$ cd src/platforms/esp32
 
-Start by updating the configuration of local `sdkconfig` file via the `menuconfig` Make target:
+Start by updating the default build configuration of local `sdkconfig` file via the `idf.py reconfigure` command:
 
-	shell$ make menuconfig
+	shell$ idf.py set-target esp32
+    shell$ idf.py reconfigure
```

Comment:
indentation

Reply:
Good catch... My editor had converted the tab to spaces. ;)